soc/xtensa/intel_adsp/common/CMakeLists.txt

```diff
@@ -100,6 +102,16 @@ add_custom_target(
       ${CMAKE_BINARY_DIR}/zephyr/main.mod 2>${NULL_FILE}
 )
 
+# west sign
+
+# FIXME: align with zephyr/cmake/mcuboot.cmake, maybe share some code
+
+add_custom_target(rimage_target ALL
```

Comment:
What happens if rimage is not in the path? Even the mcumgr module I believe looks for imgtool before calling west sign.